sections/footer.liquid

```diff
@@ -402,8 +402,7 @@
           "type": "text",
           "id": "heading",
           "default": "Quick links",
```

Comment:
I was wondering if we still needed to keep a default heading `Quick links` when adding the `Menu` block? We could leave it empty if we don't find added value. Although it does provide a hint of what text style this might take by default. Probably worth keeping after thinking out loud.